components/service/glean/docs/pings.md

```diff
@@ -67,13 +67,24 @@ This object contains experiments keyed by the experiment `id`. Each listed exper
 The pings glean generates are submitted to the Mozilla servers at specific paths, in order to provide
 additional metadata without the need of unpacking the ping payload. A typical submission URL looks like
 `"<server-address>/submit/<application-id>/<doc-type>/<glean-schema-version>/<ping-uuid>"`, with:
- 
+
 - `<server-address>`: the address of the server that receives the pings;
 - `<application-id>`: a unique application id, automatically detected by glean; this is the value returned by [`Context.getPackageName()`](http://developer.android.com/reference/android/content/Context.html#getPackageName());
 - `<doc-type>`: the name of the ping; this can be one of the pings available out of the box with glean, or a custom ping;
 - `<glean-schema-version>`: the version of the glean ping schema;
 - `<ping-uuid>`: a unique identifier for this ping.
 
+### Submitted headers
+A pre-defined set of headers is additionally sent along with the submitted ping:
+
+| Header | Value | Description |
+|--------|-------|-------------|
+| `Content-Type` | `application/json; charset=utf-8` | Describes the data sent to the server |
+| `User-Agent` | Defaults to `Glean/${GleanVersion} ({$Platform})`, can be overriden by user through [configuration](https://github.com/mozilla-mobile/android-components/blob/8441728f156847a58dcb25d29254512a1801b266/components/service/glean/src/main/java/mozilla/components/service/glean/config/Configuration.kt#L23) | Describes the application sending the ping using glean |
+| `Date` | `Mon, 23 Jan 2019 10:10:10 GMT+00:00` | Submission date/time in GMT/UTC+0 offset |
```

Comment:
nit:  Maybe put a label here showing this is an example of the date time format and not the actual value.